docs/interceptors.md

```diff
@@ -282,26 +288,73 @@ For reference, below is an example legacy Bitbucket `Interceptor` definition:
 apiVersion: triggers.tekton.dev/v1alpha1
 kind: EventListener
 metadata:
-  name: bitbucket-listener
+  name: bitbucket-server-listener
 spec:
   serviceAccountName: tekton-triggers-example-sa
   triggers:
-    - name: bitbucket-triggers
+    - name: bitbucket-server-triggers
       interceptors:
         - ref:
             name: "bitbucket"
           params:
             - name: secretRef
               value:
-                secretName: bitbucket-secret
+                secretName: bitbucket-server-secret
                 secretKey: secretToken
             - name: eventTypes
               value:
                 - repo:refs_changed
       bindings:
-        - ref: bitbucket-binding
+        - ref: bitbucket-server-binding
+      template:
+        ref: bitbucket-server-template
+```
+
+### Bitbucket Cloud
+
+A [Bitbucket cloud](https://support.atlassian.com/bitbucket-cloud/) is free to use and explore functionality, To know more about Bitbucket Cloud [follow](https://support.atlassian.com/bitbucket-cloud/docs/get-started-with-bitbucket-cloud/).
```

Comment:
I don't think we need to describe what Bitbucket cloud is given that we do not do it for other providers.